.script/tests/KqlvalidationsTests/CustomTables/Okta_CL.json

```diff
@@ -40,6 +40,10 @@
     {
       "Name": "actor_alternateId_s",
       "Type": "String"
+    },
+    {
+      "Name": "target_s",
```

Comment:
why schema having column names like columns in the log analytics table. Custom table should have column with final field names.